include/sys_matrix.h

```diff
@@ -51,7 +53,7 @@ void Matrix_SetTranslateRotateYXZ(f32 x, f32 y, f32 z, Vec3s* rot);
 
 Mtx* Matrix_MtxFToMtx(MtxF* src, Mtx* dest);
 Mtx* Matrix_ToMtx(Mtx* dest);
-Mtx* Matrix_NewMtx(struct GraphicsContext* gfxCtx);
+Mtx* Matrix_Finalize(struct GraphicsContext* gfxCtx);
 Mtx* Matrix_MtxFToNewMtx(MtxF* src, struct GraphicsContext* gfxCtx);
```

Comment:
This might be more a Dragorn question (since he did this for OoT).
Should we rename `Matrix_MtxFToNewMtx` as well to something like `Matrix_FinalizeMtxF` or `Matrix_MtxFFinalize`? There is also a `SkinMatrix` variant that would probably also need the same treatment.

Reply:
Probably not, as "finalize" refers to finalizing a transform after a succession of operations on the matrix stack, and there's no stack involved in these functions taking an external MtxF

If we really really want the symmetry (but I don't think it's needed), FinalizeExt may be a better name but meh imo